Dockerfile.repo

```diff
@@ -54,14 +53,14 @@ RUN \
     --with-apreq2-config=/usr/bin/apreq2-config && \
   make && \
   make install
-  
-RUN \
-  chkconfig httpd off
```

Comment:
This errored and is not necessary as docker won't be running init.d scripts anyway.